apps/commerce-api/src/test/kotlin/com/loopers/domain/like/LikeConcurrencyTest.kt

```
@@ -0,0 +1,166 @@
+        val latch = CountDownLatch(numberOfThreads)
+        val executor = Executors.newFixedThreadPool(numberOfThreads)
+        val successCount = AtomicInteger(0)
+        val duplicateCount = AtomicInteger(0)
+
+        // when: 같은 사용자가 동시에 좋아요 요청 (멱등성 테스트)
+        repeat(numberOfThreads) {
+            executor.submit {
+                try {
+                    likeService.addLike(userId, productId)
+                    successCount.incrementAndGet()
+                } catch (e: org.springframework.dao.DataIntegrityViolationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)